.github/workflows/build-deploy.yaml

```diff
@@ -20,27 +27,41 @@ jobs:
       repository: container-builder-template     
 
     strategy:
-
+      # Even if one docker container build fails we want the others to continue.
+      fail-fast: false
       # A matrix of Dockerfile paths and associated tags
       # Dockerfile in root builds to tag latest
       matrix:
         dockerfile: [[Dockerfile, latest], [subfolder/Dockerfile, subfolder]]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Build ${{ matrix.dockerfile[1] }}
+
+      - name: Set Container Tag Normal
         run: |
           container="${{ env.registry }}/${{ env.username}}/${{ env.repository }}:${{ matrix.dockerfile[1]}}"
-          printf "Preparing to build ${container} from ${{ matrix.dockerfile[0] }}"
+          echo "container=${container}" >> $GITHUB_ENV
+
+      #   # On a new release create a container with the same tag as the release.
+      # - name: Set Container Tag Release
+      #   if: github.event_name == 'release'
+      #   run: |
+      #     container="${{ env.registry }}/${{ env.username}}/${{ env.repository }}:${GITHUB_REF##*/}-${{ matrix.dockerfile[1]}}"
+      #     echo "container=${container}" >> $GITHUB_ENV
```

Comment:
Now that we've added the github ref and dockerfile, would it hurt to uncomment this?